plugins/nf-hello/src/main/nextflow/hello/HelloExtension.groovy

```diff
@@ -0,0 +1,78 @@
+package nextflow.hello
+
+import groovy.util.logging.Slf4j
+import groovyx.gpars.dataflow.DataflowReadChannel
+import groovyx.gpars.dataflow.DataflowWriteChannel
+import groovyx.gpars.dataflow.expression.DataflowExpression
+import nextflow.Channel
+import nextflow.Global
+import nextflow.Session
+import nextflow.extension.ChannelExtensionPoint
+import nextflow.extension.CH
+import nextflow.NF
+import nextflow.extension.DataflowHelper
+import nextflow.plugin.Scoped
+
+import java.util.concurrent.CompletableFuture
+
+/**
+ * @author : jorge <jorge.aguilera@seqera.io>
```

Comment:
Let's put some comment, especially here because this is meant to be read by newbie user who wants to learn how to write plugins